### PR TITLE
refs #40: install ldap module

### PR DESCRIPTION
**Pull request by noemi-mancini**
What the PR does:
* install ldap module (version 7.1.11 and up)
* enable it when `LDAP_ENABLE` variable is set to 1

### CODE REVIEW

File: 7.1.11-fpm-alpine3.4/Dockerfile [hunk 1]
```diff
@@ -48,6 +48,11 @@ ENV MAILHOG_VERSION v0.1.9
 RUN curl -fSL "https://github.com/mailhog/MailHog/releases/download/${MAILHOG_VERSION}/MailHog_linux_amd64" -o /usr/local/bin/mailhog \
     && chmod +x /usr/local/bin/mailhog
 
+# Install ldap module.
+RUN apk add --no-cache ldb-dev libldap openldap-dev
+RUN docker-php-ext-configure ldap
+RUN docker-php-ext-install ldap
+
```
Comment (paolomainardi) [inline 7.1.11-fpm-alpine3.4/Dockerfile]: I think we can just reuse what we are doing here https://github.com/sparkfabrik/docker-php-base-image/blob/94d2845c8af49a50677b8c8a69c2520eededd8d1/7.1.11-fpm-alpine3.4/Dockerfile#L29 and add `openldap-dev` to PHP_EXTRA_DEPS line 25.

Another thing that is missing is the `ldap.ini` file, it should part of this repo.

I would also add a test to check that is enabled/disabled.

Comment (noemi-mancini) [reply 7.1.11-fpm-alpine3.4/Dockerfile]: Ok great! The file `ldap.ini` has been added to `config.disabled` folder in each image folder.

File: 7.1.11-fpm-alpine3.4/Dockerfile [hunk 2]
```diff
@@ -48,6 +48,11 @@ ENV MAILHOG_VERSION v0.1.9
 RUN curl -fSL "https://github.com/mailhog/MailHog/releases/download/${MAILHOG_VERSION}/MailHog_linux_amd64" -o /usr/local/bin/mailhog \
     && chmod +x /usr/local/bin/mailhog
 
+# Install ldap module.
+RUN apk add --no-cache ldb-dev libldap openldap-dev
+RUN docker-php-ext-configure ldap
```
Comment (Monska85) [inline 7.1.11-fpm-alpine3.4/Dockerfile:53]: The `docker-configure-extension ldap` command creates the file `/usr/local/etc/php/conf.d/docker-php-ext-ldap.ini` which enables the extension by default. If you want to keep this extension optional you have to remove this command.

Comment (paolomainardi) [inline 7.1.11-fpm-alpine3.4/Dockerfile]: so we just need the `docker-php-ext-install` right ?

Comment (Monska85) [reply 7.1.11-fpm-alpine3.4/Dockerfile]: We need to execute both commands. We decide to move the logic into the `docker-entrypoint.sh` removing the `.ini` file if the extension should be disabled.

**Review by Monska85 — APPROVED** [review]
LGTM

**Comment by paolomainardi** [comment]
Thanks for your contribution @noemi-mancini 👍

**Pull request merged**